tests/fixtures.ts

```diff
@@ -121,6 +157,28 @@ export const test = baseTest.extend<TestFixtures, WorkerFixtures>({
 
   mcpBrowser: ['chrome', { option: true }],
 
+  mcpExtension: [false, { option: true }],
+
+  mcpExtensionPage: async ({ mcpExtension, mcpHeadless }, use) => {
+    if (!mcpExtension)
+      return await use(undefined);
```

Comment:
throw an error instead, then you don't have to assert() on it on the calling side

Reply:
Changed to `startMcpExtension`.